coalib/output/Logging.py

```diff
@@ -60,15 +62,19 @@ def configure_logging():
         'formatters': {
             'colored': {
                 '()': 'colorlog.ColoredFormatter',
-                'format': '%(log_color)s[%(levelname)s]%(reset)s[%(asctime)s]'
-                          ' %(message)s',
+                'format': '%(log_color)s[%(levelname)s]%(reset)s'
+                          '[%(asctime)s] %(message)s',
```

Comment:
This is just to make line length <80. I didn't intend to change the layout.
Will change it back to how it was.